arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dtsi

```diff
@@ -142,22 +143,15 @@
 	leds {
 		compatible = "gpio-leds";
 		pinctrl-names = "default";
-		pinctrl-0 = <&power_led_gpio>, <&status_led_gpio>;
+		pinctrl-0 = <&user_led2>;
 
-		power-status {
-			label = "power";
-			gpios = <&gpio3 RK_PD4 GPIO_ACTIVE_HIGH>;
-			linux,default-trigger = "default-on";
-			status = "disabled";
-		};
-
-		system-status {
-			label = "status";
+		user_led2 {
```

Comment:
Why only user_led2 here? The left-side code had both LEDs.

Reply:
> Why only user_led2 here? The left-side code had both LEDs, so this seems like a step back.

power(green) LED is not controllable on ROCK 4A/B/C/A+/B+.

Reply:
Ah yes, I see the schematic has it connected to +5V.
(then again the schematic also says both LEDs are green,